### ISSUE TITLE
Variations and attributes stuck on loading none stop 

### ISSUE THREAD

**Issue by lordzelos** [woocommerce/woocommerce#14744]
Having Problem with woocommerce products variations when adding variations or even editing it in products would just stuck on the spinning ball loading icon, it also happen when adding attributes.

It was working fine before and then just suddenly stopped working.

Already tried deactivating all other plugins and it still continue.

![screen1](https://cloud.githubusercontent.com/assets/27006211/25471613/200aa1a0-2b52-11e7-9e85-9695b963d50d.jpg)

**Comment by mikejolley** [comment]
Pretty sure it's a conflict but if you mail me a login I will check; mike.jolley at automattic.com

**Comment by prom9** [comment]
Hi I am having same issues with Variable products. Whenever I am trying to edit my variable products, it just keep loading continuously. This issues was encountered after I updated new version of Woocommerce.

Your quick response is much appreciated.
![variable products](https://cloud.githubusercontent.com/assets/28096092/25479997/3c4a4d06-2b8a-11e7-811b-543dae9c1589.PNG)
.

**Comment by mikejolley** [comment]
@prom9 Need access, see above.

**Comment by mikejolley** [comment]
@prom9 @lordzelos @phylynn35 I'll need access to these sites to debug, otherwise I cannot fix this as it doesn't happen on my test stores.

**Comment by lordzelos** [comment]
@mikejolley i emailed you the login to wordpress.

**Comment by mikejolley** [comment]
@lordzelos nothing came. Try the email on my profile.

**Comment by lordzelos** [comment]
@mikejolley ok just tried  using your email on your profile.

**Comment by phylynn35** [comment]
To make sure I get this correct. You need access to my Word Press website.
Correct?  If so I will set up a temporary password for you.

**Comment by mikejolley** [comment]
Yeah I'd like to take a look. @lordzelos has an issue where it's blocking access with 403 error, and I think thats security related and blocking ajax too. Server issue.

**Comment by stephmadfx** [comment]
Hello I have the same problem !
Can you look again?

**Comment by karlarmoura** [comment]
Same problem here! Any solutions? I'm lost, any help is welcome.

**Comment by WordCommerce** [comment]
Hi,
I had the same problem!
I disabled the yoast seo plugin and eventually edited the product.

**Comment by relsi1053** [comment]
I have the same problem
But I didn't find any solution

**Comment by karlarmoura** [comment]
Found out what was the problem for me, but not a very good solution:
It's an incompatibility with the plugin "Ultimate Addons for Visual Composer". So when I want do edit the products, I need to deactivate this plugin. It's bad because it mess with the entire pages. Still following here for a efficient solution.

**Comment by brhyne** [comment]
I'm having the same issue too. Additionally, I have "Ultimate Addons for Visual Composer" as well, but deactivating it did not resolve my issue.  Any help would be greatly appreciated!

Thanks in advance!

`
Uncaught Error: Option 'ajax' is not allowed for Select2 when attached to a <select> element.
    at String.<anonymous> (select2.js:925)
    at Function.each (load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:2)
    at constructor.prepareOpts (select2.js:923)
    at constructor.prepareOpts (select2.js:2581)
    at constructor.init (select2.js:690)
    at HTMLSelectElement.<anonymous> (select2.js:3372)
    at Function.each (load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:2)
    at a.fn.init.each (load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:2)
    at a.fn.init.$.fn.select2 (select2.js:3359)
    at HTMLSelectElement.<anonymous> (wc-enhanced-select.min.js:1)
(anonymous) @ select2.js:925
each @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:2
prepareOpts @ select2.js:923
prepareOpts @ select2.js:2581
init @ select2.js:690
(anonymous) @ select2.js:3372
each @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:2
each @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:2
$.fn.select2 @ select2.js:3359
(anonymous) @ wc-enhanced-select.min.js:1
each @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:2
each @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:2
(anonymous) @ wc-enhanced-select.min.js:1
dispatch @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:3
r.handle @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:3
trigger @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:3
a.event.trigger @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:9
(anonymous) @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:3
each @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:2
each @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:2
trigger @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:3
variations_loaded @ meta-boxes-product-variation.min.js:1
dispatch @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:3
r.handle @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:3
trigger @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:3
a.event.trigger @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:9
(anonymous) @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:3
each @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:2
each @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:2
trigger @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:3
success @ meta-boxes-product-variation.min.js:1
i @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:2
fireWith @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:2
y @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:4
c @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,jquery-ui-core,jquery-ui-widget,jquery-ui-mo…:4



`

**Comment by GoosePT** [comment]
Hello All!

Same here. YOAST SEO and VC disable did not solve the problem. :(

**Comment by lordzelos** [comment]
I just found out that it was Ultimate Visual composer addons plugins that was effecting the loading icons, it seems to work fine for me now if i don't use that plugins.

**Comment by mikejolley** [comment]
Please see this thread regarding Select2 conflicts:

https://github.com/woocommerce/woocommerce/wiki/Select2-fields-not-working-in-3.0.x

TL;DR plugins need to either not enqueue old select2 on WC pages, or use Select2 v4. I'm going to lock the comments on this issue so people see the link I just posted instead of 'me too' replies. 

As for visual composer, the author says he has released an update, so update it. I cannot test this because it's premium.

Thanks!
